lib/tasks/migrate.rake

```diff
@@ -1,11 +0,0 @@
-namespace :curation_concerns do
-  namespace :migrate do
-    desc "Migrate audit logs"
-    task audit_logs: :environment do
```

Comment:
Nope, it's here: https://github.com/projecthydra/sufia/blob/master/tasks/migrate.rake#L14